src/main/resources/role-permission.yml

```diff
@@ -34,4 +34,4 @@ rolePerms:
     - /monitor/*.query
     - /consumer/*.query
     - /cluster/*.query
-    - /dlqMessage/*.query
+    - /dlqMessage/**
```

Comment:
How about using this.
```
    - /dlqMessage/*.query
    - /dlqMessage/exportDlqMessage.do
    - /dlqMessage/batchExportDlqMessage.do
```

Because there may be write operations in this controller, e.g. `/batchResendDlqMessage.do`, ordinary users should not have write permission by default.